Library/Formula/dlite.rb

```diff
@@ -0,0 +1,94 @@
+require "language/go"
+
+class Dlite < Formula
+  desc "The simplest way to use Docker on OSX"
+  homepage "https://github.com/nlf/dlite"
+  url "https://github.com/nlf/dlite/archive/1.1.1.tar.gz"
+  sha256 "e63b81be5fec5b3b55e3b364f4bc77cfd3ded3920a4cc124431a2680e7f46178"
+
+  # DLite depends on xhyve which only works on OSX versions 10.10 (Yosemite) or newer
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+
+  go_resource "github.com/TheNewNormal/libxhyve" do
```

Comment:
i've actually vendored all the dependencies, so i think most of this can be removed